==== triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h#12 - /google/src/cloud/wcui/mlir_b126ee65fcbb49054e32fd11fdac07279d00f159_1687853092/triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h ====
# action=edit type=text
--- triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h	2023-06-01 05:23:23.000000000 -0700
+++ triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h	2023-06-27 01:11:28.000000000 -0700
@@ -129,7 +129,7 @@
     }
     auto newFuncOp = rewriter.create<LLVM::LLVMFuncOp>(
         funcOp.getLoc(), funcOp.getName(), llvmType, linkage,
-        /*dsoLocal*/ false, LLVM::CConv::C, attributes);
+        /*dsoLocal*/ false, LLVM::CConv::C, /*comdat=*/nullptr, attributes);
     rewriter.inlineRegionBefore(funcOp.getBody(), newFuncOp.getBody(),
                                 newFuncOp.end());
     if (failed(rewriter.convertRegionTypes(&newFuncOp.getBody(), *typeConverter,
